### PR TITLE
Dt/feature/post-controller-update

### PR DESCRIPTION
**Pull request by DaTanUmanitoba**
modify the GET method of post-controller to the correct format, and paying debts for the related tests

add new data to the res body

### CODE REVIEW

**Review by ShahzaibParacha — APPROVED** [review]

File: backend/controller/post-controller.js [hunk 1]
```diff
@@ -84,43 +74,42 @@ const removePostByID = async (req, res) => {
...
+    }
+}
+
+async function getPostsInfo(posts){
+    let result = []
+    for( let post of posts){
+        const user = await userService.getUserInfo(post.user_id)
+        const likes = await likeService.getNumLikes(post._id)
+        if( user !== null ){
+            result.push({username: user.username, affiliation: user.affiliation, numberLikes: likes})
```
Comment (orangeboy55555) [inline backend/controller/post-controller.js:109]: I think Shahzaib wants the profile picture as well.

**Review by orangeboy55555 — CHANGES_REQUESTED** [review]

File: backend/controller/post-controller.js (hunk 1 above)
Comment (ShahzaibParacha) [reply backend/controller/post-controller.js:109]: true

**Review by orangeboy55555 — APPROVED** [review]
Looks good

**Review by ShahzaibParacha — APPROVED** [review]
looks good now

**Pull request merged**